lib/usdUfe/ufe/StagesSubject.cpp

```diff
@@ -157,13 +157,19 @@ void sendAttributeChanged(
                 notifyWithoutExceptions<Ufe::Camera>(ufePath);
             }
         } else {
+            // Special case when Redo-ing visibility change, the notice.GetChangedInfoOnlyPaths()
+            // does not contain the change, hence handling visibility notification in re-sync path.
+            if (changedToken == UsdGeomTokens->visibility) {
+                Ufe::VisibilityChanged vis(ufePath);
+                notifyWithoutExceptions<Ufe::Object3d>(vis);
+            }
```

Comment:
This notif is only within the ifdef V4 block, which means it won't be sent for Maya 2022/2023. That is why the test is failing.

Reply:
Thanks for catching that.